src/util/initEvents.native.ts

```diff
@@ -0,0 +1,134 @@
+import _ from 'lodash'
+import { Store } from 'redux'
+import { inputHandlers /* isGestureHint */ } from '../shortcuts'
+import * as db from '../data-providers/dexie'
+import { clearSelection, isRoot, pathToContext } from '.'
+import { State } from './initialState'
+import { decodeThoughtsUrl, pathExists } from '../selectors'
+import { /* alert */ error, scrollCursorIntoView, setCursor, toggleTopControlsAndBreadcrumbs } from '../action-creators'
```

Comment:
FYI the `alert` action-creator is not a browser/native alert but an in-app alert system, as you're probably aware. The only web dependency it has is `window.setTimeout`.

Reply:
Thanks @raineorshine.